chart/kubeapps/README.md

```diff
@@ -153,6 +153,104 @@ If you have dedicated a namespace only for Kubeapps you can completely clean the
+
+# Make Kubeapps accesible from your local address
+kubectl port-forward -n kubeapps svc/kubeapps 8080:80
+
+# Visit http://127.0.0.1:8080 in you browser and enter the token you obtained before.
+```
+
+### How to install Kubeapps in production scenarios?
+
+For any user-facing installation you should [configure an OAuth2/OIDC provider](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md) to enable secure user authentication with Kubeapps and the cluster.
+Please also refer to the [Access Control](https://github.com/kubeapps/kubeapps/blob/master/docs/user/access-control.md) documentation to configure fine-grained access control for users.
+
+### How to use Kubeapps?
+
+Have a look to [this guide](https://github.com/kubeapps/kubeapps/blob/master/docs/user/dasboard.md) for knowing how to use the Kubeapps dashboard.
+
+### How to configure Kubeapps with Ingress
+
+The example below will match the URL `http://example.com` to the Kubeapps dashboard. For further configuration, please refer to your specific Ingress configuration docs (e.g., [NGINX](https://github.com/kubernetes/ingress-nginx) or [HAProxy](https://github.com/haproxytech/kubernetes-ingress)).
+
+```bash
+helm install kubeapps --namespace kubeapps \
+  --set ingress.enabled=true \
+  --set ingress.hostname=example.com \
+    bitnami/kubeapps
+```
+#### Serving Kubeapps in a subpath
+
+Eventually, you may want to serve Kubeapps with a subpath, for instance `http://example.com/subpath`. If you are using the ingress configuration provided by the Kubeapps chart, note that currently it does not support customizing that `path` value, so, consequently you have to manually patch the ingress object:
+
+```bash
+kubectl patch ingress kubeapps -n kubeapps --type=json -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/path", "value":"/subpath"}]'
+```
+
+Besides, if you are using the OAuth2/OIDC login (more information in [this guide](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md)), you will need, also, to configure the different URLs:
+
+```bash
+helm install kubeapps bitnami/kubeapps \
+  --namespace kubeapps \
+  # ... other OIDC flags 
+  --set authProxy.oauthLoginURI="/subpath/oauth2/login" \
+  --set authProxy.oauthLogoutURI="/subpath/oauth2/logout" \
+  --set authProxy.additionalFlags="{<other flags>,--proxy-prefix=/subpath/oauth2}"
+```
+
+### Can Kubeapps be installed in more than one cluster?
+
+Yes! Kubeapps 2.0+ supports multicluster environments. Check out [this guide](https://github.com/kubeapps/kubeapps/blob/master/docs/user/deploying-to-multiple-clusters.md) to know more.
+
+### How to install Kubeapps without Internet connection?
+
+Yes! Follow [this guide](https://github.com/kubeapps/kubeapps/blob/master/docs/user/offline-installation.md) to discover how to perform an installation in an air-gapped scenario.
```

Comment:
If you want to start with "Yes!", perhaps update the title to a "Can Kubeapps be installed without ..."?